In [1]:
import pandas as pd
import os

In [2]:
os.chdir("/ufrc/soltis/shan158538/TragFL_NewAnalysis/OutPut/DE_analysis/Additive_expression_Tms/")

In [3]:
## Input files
DESeq2_result = pd.read_table("DESeq2_Tms_nonAdditive_loci.txt", sep = "\t")

In [10]:
DESeq2_result.shape

(1768, 6)

In [4]:
DESeq2_result.head()

baseMean    ...         padj
Tpr_TRINITY_DN1451_c0_g1|Tdu_TRINITY_DN16990_c3_g1  100.236819    ...     0.001337
Tpr_TRINITY_DN10853_c1_g4|Tdu_TRINITY_DN21860_c...  750.334395    ...     0.006242
Tpr_TRINITY_DN12798_c2_g2|Tdu_TRINITY_DN24179_c...   58.686028    ...     0.001516
Tpr_TRINITY_DN8521_c0_g1|Tdu_TRINITY_DN16367_c0_g1    4.080691    ...     0.048222
Tpr_TRINITY_DN11678_c3_g2|Tdu_TRINITY_DN24143_c...   57.367802    ...     0.000751

[5 rows x 6 columns]

In [5]:
Limma_result = pd.read_table("DE_additive_Tdu_Tpr-Tms_sig_loci.txt", sep = ",")

In [11]:
Limma_result.shape

(1312, 7)

In [6]:
Limma_result.head()

Unnamed: 0    ...            B
0  Tpr_TRINITY_DN12798_c2_g2|Tdu_TRINITY_DN24179_...    ...     0.376765
1  Tpr_TRINITY_DN11678_c3_g2|Tdu_TRINITY_DN24143_...    ...     0.124218
2  Tpr_TRINITY_DN13683_c4_g1|Tdu_TRINITY_DN22426_...    ...    -1.330421
3  Tpr_TRINITY_DN13082_c0_g7|Tdu_TRINITY_DN20860_...    ...    -0.662058
4  Tpr_TRINITY_DN14614_c1_g6|Tdu_TRINITY_DN1401_c...    ...    -1.350833

[5 rows x 7 columns]

In [7]:
Shared = pd.merge(DESeq2_result, Limma_result, how='inner', left_index=True, right_on=["Unnamed: 0"])

In [8]:
Shared.shape

(1013, 13)

In [9]:
Shared.head()

baseMean  log2FoldChange     lfcSE    ...      P.Value  adj.P.Val         B
0  58.686028        0.962326  0.245485    ...     0.000311   0.009613  0.376765
1  57.367802        1.781473  0.431114    ...     0.000469   0.011362  0.124218
2  20.461439        0.844411  0.290551    ...     0.002141   0.024171 -1.330421
4  69.575222       -1.127549  0.298756    ...     0.001959   0.023035 -1.350833
5  18.403122        1.404077  0.390851    ...     0.000313   0.009613  0.548669

[5 rows x 13 columns]

### In Tms, the number of none-additive loci that are identified by both DESeq2 and Limma is 1,013
### DESeq2 identified 1,768 none-additive loci
### Limma identified 1,312 none-additive loci